### PR TITLE
In range loading animation #27, blank message screen #31, Rightmesh logo cut off #17

### PR DESCRIPTION
**Pull request by ghost**
This PR adds an animation in the range tab, adds an empty view for listview in message tab, and fixes the Rightmesh logo being cut off in certain devices in the splash screen

The in range animation is not quite right yet but can be fixed anytime we have the right animation

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -63,7 +92,49 @@
...
+                        app:layout_constraintTop_toBottomOf="@+id/imageView" />
+
+                    <TextView
+                        android:id="@+id/textView8"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_marginEnd="@dimen/meshim_margin_small"
+                        android:layout_marginStart="@dimen/meshim_margin_small"
+                        android:layout_marginTop="@dimen/meshim_margin_small"
+                        android:text="@string/emty_message_text2"
```
Comment (fractalic) [inline app/src/main/res/layout/activity_main.xml]: emty -> empty

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -63,7 +92,49 @@
...
+                        android:layout_height="wrap_content"
+                        android:layout_marginTop="@dimen/meshim_margin_small"
+                        android:text="@string/empty_message_text1"
+                        android:textStyle="bold"
+                        app:layout_constraintEnd_toEndOf="parent"
+                        app:layout_constraintStart_toStartOf="parent"
+                        app:layout_constraintTop_toBottomOf="@+id/imageView" />
+
+                    <TextView
+                        android:id="@+id/textView8"
```
Comment (fractalic) [inline app/src/main/res/layout/activity_main.xml]: could make the id tags more descriptive for these views (id should describe the purpose of the field)

File: app/src/main/java/io/left/meshim/activities/MainActivity.java [hunk 3]
```diff
@@ -185,6 +185,7 @@ private void configureUserList() {
      */
     private void configureMessageList() {
         ListView listView = findViewById(R.id.multiUserMessageListView);
+        listView.setEmptyView(findViewById(R.id.empty_message_view));
```
Comment (fractalic) [inline app/src/main/java/io/left/meshim/activities/MainActivity.java]: snake_case inconsistent with camelCase for other R.id's

**Review by fractalic — CHANGES_REQUESTED** [review]
quibbles

File: app/src/main/java/io/left/meshim/activities/MainActivity.java [hunk 4]
```diff
@@ -167,7 +167,7 @@ public void onTabChanged(String s) {
      */
     private void configureUserList() {
         ListView listView = findViewById(R.id.userListView);
-        listView.setEmptyView(findViewById(R.id.empty_list_item));
+        listView.setEmptyView(findViewById(R.id.emptyListItem));
```
Comment (FrazerSeymour) [inline app/src/main/java/io/left/meshim/activities/MainActivity.java]: The Google convention for resources is `snake_case`.

File: app/src/main/res/layout/activity_main.xml [hunk 5]
```diff
@@ -37,17 +38,45 @@
...
-                    app:layout_constraintEnd_toStartOf="parent"
+                    app:layout_constraintBottom_toBottomOf="parent"
+                    app:layout_constraintEnd_toEndOf="parent"
                     app:layout_constraintStart_toStartOf="parent"
-                    app:layout_constraintTop_toTopOf="parent" />
-            </android.support.constraint.ConstraintLayout>
+                    app:layout_constraintTop_toTopOf="parent">
+
+                    <com.airbnb.lottie.LottieAnimationView
+                        android:id="@+id/emptyListItem1"
```
Comment (FrazerSeymour) [inline app/src/main/res/layout/activity_main.xml]: An id that indicated what this is would be useful, e.g. `user_list_loading_item`, `user_list_loading_animation`, `user_list_loading_text`. These IDs are kind of the equivalent of `String string1; String string2; int num;`.

**Review by FrazerSeymour — APPROVED** [review]
ID nitpicks aside, logic looks fine. :+1:

**Comment by FrazerSeymour** [comment]
Oh, just seeing that @fractalic left similar comments GitHub just marked them as outdated. :upside_down_face:

File: app/src/main/res/layout/activity_main.xml [hunk 6]
```diff
@@ -113,27 +112,28 @@
...
 
                     <TextView
-                        android:id="@+id/emptyMessageText1"
+                        android:id="@+id/message_list_loading_text1"
                         android:layout_width="wrap_content"
                         android:layout_height="wrap_content"
                         android:layout_marginEnd="@dimen/meshim_margin_small"
                         android:layout_marginStart="@dimen/meshim_margin_small"
-                        android:layout_marginTop="@dimen/meshim_margin_small"
+                        android:layout_marginTop="8dp"
```
Comment (FrazerSeymour) [inline app/src/main/res/layout/activity_main.xml]: WYSIWYG editor snuck a magic number in again.

Comment (ghost) [reply app/src/main/res/layout/activity_main.xml]: fixed it!

**Pull request merged**